pyqtgraph/functions.py

```diff
@@ -947,73 +951,48 @@ def rescaleData(data, scale, offset, dtype=None, clip=None):
     else:
         dtype = np.dtype(dtype)
     
-    try:
-        if not getConfigOption('useWeave'):
-            raise Exception('Weave is disabled; falling back to slower version.')
-        try:
-            import scipy.weave
-        except ImportError:
-            raise Exception('scipy.weave is not importable; falling back to slower version.')
-        
-        ## require native dtype when using weave
-        if not data.dtype.isnative:
-            data = data.astype(data.dtype.newbyteorder('='))
-        if not dtype.isnative:
-            weaveDtype = dtype.newbyteorder('=')
+    # p = np.poly1d([scale, -offset*scale])
```

Comment:
can we remove commented out lines?

Reply:
I didn't know if this project had a strong preference to keep old bits of code like this around, but I would be happy to drop unused comments.

Reply:
I don't mind leaving some old bits like this in, but probably should add a note as to why you're wanting to keep them around